dev-libs/libstrophe/libstrophe-0.9.2.ebuild

```diff
@@ -0,0 +1,21 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit autotools
+
+DESCRIPTION="A simple, lightweight C library for writing XMPP clients"
+HOMEPAGE="http://strophe.im/libstrophe"
+SRC_URI="https://github.com/strophe/libstrophe/releases/download/${PV}/${P}.tar.gz"
+
+LICENSE="GPL-3 MIT"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE=""
+
+DEPEND="dev-libs/openssl:0=
+	dev-libs/expat"
+
+RDEPEND="${DEPEND}"
+BDEPEND="${DEPEND}"
```

Comment:
Nope, this is wrong. BDEPEND is a split from DEPEND, so their contents can't be the same :) BDEPEND are executable build-dependencies, like doxygen, gettext and pkg-config.

Reply:
Gotcha, so it should be `BDEPEND=""`?

Reply:
In this state, yes. However you will most likely want to add a doc use flag with doxygen dependency. Then it'd be `BDEPEND="doc? ( app-doc/doxygen )"`